docs/technical-reports/survivable-eclipse-durations/model.md

```diff
@@ -0,0 +1,91 @@
+
+Random variables.
+
+- Let t1 be a random variable that is the age of C as of the onset of the eclipse.
+- Let d be a random variable that is the maximum length of all short forks that branch off of C and end with an honest block.
+- Let t2 be a random variable that is how long it takes after the onset of the eclipse for the adversary to have been elected in `k - d` slots after d+tipSlot(C).
+
+We desire a bound Pr(BAD) < EPSILON where BAD is the event in which the adversary succeeds despite t2 < CUTOFF for some sufficiently long duration CUTOFF and a sufficiently low probability EPSILON.
+
+We know the following, for a 1/3 adversary.
+
+- (1) Assuming all honest stake is online and caught-up, details of the Praos election mechanism ensure Pr(t1 > X) <= e^(-2 (k - mu(X) - 1)^2 / X).
+  See the appendix below for the derivation.
+- (2) The probability Pr(d >= Y) has an upper bound approximated by e^(-0.0716 - 0.0523 Y).
+  TODO explanation/citation from Alexander Russell (and Peter Gaži?)
+- (3) The probability Pr(BAD | t1=x, d=y) has an upper bound of NegativeBinomialTrials(k - y, phi(0.33)).cdf(CUTOFF + x - y).
+  The subtraction of y from CUTOFF is pessimitically assuming that the short fork of length d of perfectly dense.
+
+Consider the following expansion of Pr(BAD) via the law of total probability with respect to the event t1 < X && d < Y.
+
+- Pr(BAD) = Pr(t1 >= X || d >= Y) Pr(BAD | t1 >= X || d >= Y) + Pr(t1 < X, d < Y) Pr(BAD | t1 < X, d < Y)
+
+If the stronger Pr(t1 >= X || d >= Y) + Pr(BAD | t1 < X, d < Y) <= EPSILON holds, then we definitely have Pr(BAD) <= EPSILON, regardless of how much smaller than 1 the probatilites Pr(BAD | t1 >= X || d >= Y) and Pr(t1 < X, d < Y) actually are, which is useful since we haven't yet established upper bounds for those.
+
+Lett u and v abbreviate the upper bounds (1) and (2) above, so that we have Pr(t1 >= X) <= u and Pr(d >= Y) <= v.
+This implies Pr(t1 >= X || d >= Y) <= 1 - (1 - u)(1 - v) (which simplifies to u + v - uv, FYI).
+
+Finally, since Pr(BAD) is covariant with respect to t1 and also with respect to d (since its occurrence in k - y dominates its occurrence in the cdf argument), we have Pr(BAD | t1 < X, d < Y) <= Pr(BAD | t1=X-1, d=Y-1), ie the uppper bound (3) above.
+
+## Conclusion
+
+Thus, if we can find coherent values of X, Y, CUTOFF, and EPSILON, then an eclipse lasting less than CUTOFF will defeat the victim only with a probability less than EPSILON, unless the adversary has more than 1/3 stake or else too much of the honest stake not online, caught-up, and Delta-well-connected.
+
+## Appendix 1: Bounding age of immutable tip
+
+Let GR_W be a random variable that counts the minimum possible growth of the honest chain within some fixed interval of W slots (ie the growth assuming every message was delayed by the full Delta).
+Note that 0 <= GR_W - GR_{W-1} <= 1 and 0 <= GR_{W+1} - GR_W <= 1.
+(TODO This seems right to me, but probably deserves a proof sketch.)
```

Comment:
TODO

Reply:
Note to self: Alexander Russell sketched this in a private Slack message. https://input-output-rnd.slack.com/archives/C02EF6ET4DC/p1697038571622419?thread_ts=1696962923.870129&cid=C02EF6ET4DC